pages branch: 'IViewComponentHelper' does not contain a definition for 'Invoke' and no extension method 'Invoke' accepting a first argument of type 'IViewComponentHelper'

Error when running on non-windows

```
Microsoft.AspNetCore.Mvc.Razor.Compilation.CompilationFailedException: One or more compilation failures occurred:
/Views/Shared/Post.cshtml(5,43): error CS1061: 'IViewComponentHelper' does not contain a definition for 'Invoke' and no extension method 'Invoke' accepting a first argument of type 'IViewComponentHelper' could be found (are you missing a using directive or an assembly reference?)
/Views/Shared/Post.cshtml(6,43): error CS1061: 'IViewComponentHelper' does not contain a definition for 'Invoke' and no extension method 'Invoke' accepting a first argument of type 'IViewComponentHelper' could be found (are you missing a using directive or an assembly reference?)
   at Microsoft.AspNetCore.Mvc.Razor.Compilation.CompilationResult.EnsureSuccessful()
   at Microsoft.AspNetCore.Mvc.Razor.Internal.CompilerCache.CreateCacheEntry(String relativePath, String normalizedPath, Func`2 compile)
```

--- a/src/Views/Shared/Page.cshtml
+++ b/src/Views/Shared/Page.cshtml
@@ -1,8 +1,8 @@
 @model downr.Models.Metadata
 
 @section Head {
-    @Component.Invoke("Metadata", new { metadata = Model })
-    @Component.Invoke("OpenGraph", new { metadata = Model })
+    @Component.InvokeAsync("Metadata", new { metadata = Model })
+    @Component.InvokeAsync("OpenGraph", new { metadata = Model })
 }
 
 <a id="@Model.Slug"></a>

